### PR TITLE
Object manipulation 2 | Jazzmine Jimenez

### PR DESCRIPTION
**Pull request by Jazzmine-Jimenez**
image

### CODE REVIEW

File: object-manipulation-2/src/omit.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-unused-vars */
+function omit(source, keys) {
+  var newObject = {};
+  for (var key in source) {
+    for (var i = 0; i < keys.length; i++) {
+      if (keys[i] === key) break;
+      else continue;
```
Comment (shadorki) [inline object-manipulation-2/src/omit.js]: You don't need an else continue here, the loop will continue regardless

File: object-manipulation-2/src/pick.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  var newObj = {};
+  for (var key in source) {
+    for (var i = 0; i < keys.length; i++) {
+      if (keys[i] !== key) continue;
+      else {
+        if (source[keys[i]] !== undefined) {
+          newObj[keys[i]] = source[keys[i]];
+          break;
+        }
+      }
```
Comment (shadorki) [inline object-manipulation-2/src/pick.js]: This gets very hard to read when there is a use of a continue statements.
```
if(keys[i] === key) {
         if (source[keys[i]] !== undefined) {
          newObj[keys[i]] = source[keys[i]];
          break;
        }
}

**Review by shadorki — CHANGES_REQUESTED** [review]

File: object-manipulation-2/src/pick.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars */
+function pick(source, keys) {
+  var newObj = {};
+  for (var key in source) {
+    for (var i = 0; i < keys.length; i++) {
+      if (keys[i] === key) {
```
Comment (thebearingedge) [inline object-manipulation-2/src/pick.js]: You can remove your `for` loop on line 4 and `if` statement on line 6.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Nice, just one small tweak.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**